onfido/onfido.py

```diff
@@ -7,10 +7,10 @@
 
 
 class Api(object):
-    def __init__(self, api_key, requestor=None):
+    def __init__(self, api_key, version='v2',requestor=None):
 
```

Comment:
We tend to get rid of these spaces between the function definition and the actual function code so between def __init__() and the function code